CITATION.cff

```diff
@@ -0,0 +1,72 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: ASPECT
+message: >-
+  If you use this software, please see our website for the latest citation information. https://aspect.geodynamics.org/citing.html?src=www&ver=release
+type: software
+authors:
+  - given-names: Wolfgang
+    family-names: Bangerth
+    affiliation: 'Colorado State University, USA'
+    orcid: 'https://orcid.org/0000-0003-2311-9402'
+    email: bangerth@colostate.edu
+  - given-names: 'Juliane'
+    family-names: Dannberg
+    email: judannberg@gmail.com
+    affiliation: >-
+      GEOMAR Helmholtz Centre for Ocean Research Kiel and
+      Kiel University
+    orcid: 'https://orcid.org/0000-0003-0357-7115'
+  - given-names: Menno
+    family-names: Fraters
+    email: menno.fraters@tutanota.com
+    orcid: 'https://orcid.org/0000-0003-0035-7723'
+  - given-names: Rene
+    family-names: Gassmoeller
+    affiliation: GEOMAR Helmholtz Centre for Ocean Research Kiel
+    email: rene.gassmoeller@mailbox.org
+    orcid: 'https://orcid.org/0000-0001-7098-8198'
+  - given-names: Anne
+    family-names: Glerum
+    orcid: 'https://orcid.org/0000-0002-9481-1749'
+    email: acglerum@gfz-potsdam.de
+    affiliation: 'GFZ Potsdam, Germany'
+  - given-names: Timo
+    family-names: Heister
+    email: timo.heister@gmail.com
+    affiliation: Clemson University
+    orcid: 'https://orcid.org/0000-0002-8137-3903'
+  - given-names: Bob
+    family-names: Myhill
+    email: bob.myhill@bristol.ac.uk
+    affiliation: University of Bristol
+    orcid: 'https://orcid.org/0000-0001-9489-5236'
+  - given-names: John
+    family-names: Naliboff
+    email: john.naliboff@gmail.com
+    affiliation: New Mexico Tech
+    orcid: 'https://orcid.org/0000-0002-5697-7203'
+  - {}
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.592692
+    description: concept DOI
+  - type: doi
+    value: 10.5281/zenodo.8200213
+    description: version 2.5.0
```

Comment:
I don't quite understand. What is "concept DOI"? Both DOIs link to ASPECT 2.5.0 on Zenodo, right? What's with this description: shouldn't that be the name of the article (ASPECT 2.5.0)?

Reply:
A concept doi is a link to the project (the concept of the idea if that makes any sense) as a whole and not to a specific version.